auto_submit/lib/service/revert_request_validation_service.dart

```diff
@@ -303,4 +317,39 @@ class RevertRequestValidationService extends ValidationService {
     log.info('Ack the processed message : $ackId.');
     await pubsub.acknowledge(config.pubsubRevertRequestSubscription, ackId);
   }
+
+  Future<DiscordNotification> get discordNotificationClient async {
```

Comment:
Move to `discordNotification.dart`.

Reply:
Hmm in this case I think it makes sense here since we get most of our clients from this class.